INSTALL.md

```diff
@@ -302,23 +302,24 @@ yarn install
 
 ## Setup Database
 
-Finally initialize the database with some seed data by running:
+Finally, initialize the database with some seed data by running:
 
 ```bash
 bundle exec rake db:setup
 ```
 
-You may need to set up the database from scratch again, in that case use `db:reset` to drop it first too:
+You may need to set up the database from scratch again, in that case, use `db:reset` to drop it first too:
 
 ```bash
-bundle exec rake db:reset # This will drop and setup the database
+bundle exec rake db:reset # This will drop and set up the database
 ```
 
-**NOTE:** This will seed the application and creates the Master, Provider & Developer accounts which are accessible through: `http://master-account.example.com.lvh.me:3000`, `http://provider-admin.example.com.lvh.me:3000`, `http://provider.example.com.lvh.me:3000` respectively. Please take note of the credentials generated at this moment also so that you can log into each of these portals.
+**NOTE:** This will seed the application and creates the Master, Provider & Developer accounts that are accessible through: `http://master-account.stg-saas.localhost:3000/`, `http://provider-admin.stg-saas.localhost:3000/`, `http://provider.stg-saas.localhost:3000/` respectively. 
+Note down the credentials generated at this moment, to log in to each of the portals above.
```

Comment:
In fact it should not be `stg-saas.localhost`, but `any-domain.localhost`

Reply:
Right @hallelujah , but `stg-saas.localhost` is the one in  the **example** config `config/examples/domain_substitution.yml`, as you can see:
https://github.com/3scale/porta/blob/a14d790a70c28f974f9d2a2af45575be56e64b55/config/examples/domain_substitution.yml#L1-L19

This means that it is what the people following this guide will have at this point in `config/domain_substitution.yml` because in a previous step we told them to do `cp config/examples/* config/`. Look at https://github.com/3scale/porta/blob/master/INSTALL.md#config-files.

Conclusion: IMO, the simplest approach in this line would be mentioning `stg-saas.localhost` as @srijitamukherjee did, for anyone who just wants to quickly set the Porta/System up without knowing any internals of it.

Reply:
replace `stg-saas` by something else something like `3scale.localhost` ;)

Reply:
I like more `3scale.localhost` too 😃